tests/unit/test_configuration.py

```diff
@@ -163,30 +163,30 @@ def test_loading_from_yaml(tmp_path):
 
 
 @pytest.mark.parametrize(
-    "user_config, expected",
+    "age_differences, expected",
```

Comment:
I'm honestly kinda surprised this worked since we were using `user_config` as both the parameterized age differences lists as well as the name of the dictionary.